data/archive/nwerc/1995.yaml

```diff
@@ -1,10 +1,10 @@
 contest:
   name: NWERC 1995
-  date: Amsterdam (VU)
+  date: Amsterdam (VU), 10-11 November
   type: nwercpc
   year: 1995
 resources:
-  problems: problemset.pdf # TODO first 4 problems only
+  problems: nwerc95.pdf
```

Comment:
It looks like 1995 was also a "WERC" instead of an "NWERC":

![image](https://user-images.githubusercontent.com/9739541/221366031-ddd64ff5-5544-4de5-9689-5f23b0f232c2.png)

But then the four-problem PDF that we had before, has the exact same problems for A-D, but has "NWERC" as header. :sweat_smile: 

![image](https://user-images.githubusercontent.com/9739541/221366153-2a573ec4-7498-45cf-a6a4-5d05d79389f3.png)

Reply:
This one a weirder case. I decided to go with the name as shown on the program booklet that was included in the folder.
![IMG20230226080842](https://user-images.githubusercontent.com/5221158/221397263-9ee5c800-c40e-4ca4-831f-39d98d1236e8.jpg)

Reply:
Given the preface of NWERC 1995 that you shared below, I think it should indeed be an NWERC :slightly_smiling_face: